MAP-LIB_ProjectRules.md

```diff
@@ -249,41 +249,41 @@ Members:
+  | `Modelica.ComplexBlocks`              | Anton Haumer, Christian Kral                              |
+  | `Modelica.Blocks.Tables`              | Thomas Beutlich, Martin Otter, Anton Haumer               |
+  | `Modelica.StateGraph`                 | Martin Otter, Hans Olsson                                 |
+  | `Modelica.Electrical.Analog`          | Christoph Clauss, Kristin Majetta, Christian Kral         |
+  | `Modelica.Electrical.Digital`         | Christoph Clauss, Kristin Majetta                         |
+  | `Modelica.Electrical.Machines`        | Anton Haumer, Christian Kral                              |
+  | `Modelica.Electrical.Multiphase`      | Anton Haumer, Christian Kral                              |
+  | `Modelica.Electrical.PowerConverters` | Christian Kral, Anton Haumer                              |
+  | `Modelica.Electrical.QuasiStationary` | Anton Haumer, Christian Kral                              |
+  | `Modelica.Electrical.Spice3`          | Christoph Clauss, Kristin Majetta, Joe Riel               |
+  | `Modelica.Magnetic.FluxTubes`         | Thomas Bödrich, Johannes Ziske                            |
+  | `Modelica.Magnetic.FundamentalWave`   | Christian Kral, Anton Haumer                              |
+  | `Modelica.Magnetic.QuasiStatic`       | Christian Kral, Anton Haumer                              |
+  | `Modelica.Mechanics.MultiBody`        | Jakub Tobolar, Martin Otter                               |
+  | `Modelica.Mechanics.Rotational`       | Jakub Tobolar, Martin Otter, Anton Haumer, Christian Kral |
+  | `Modelica.Mechanics.Translational`    | Anton Haumer, Jakub Tobolar, Martin Otter, Christian Kral |
+  | `Modelica.Fluid`                      | Rüdiger Franke, Francesco Casella, Hubertus Tummescheit   |
+  | `Modelica.Fluid.Dissipation`          | Stefan Wischhusen, Francesco Casella                      |
+  | `Modelica.Media`                      | Hubertus Tummescheit, Francesco Casella, Rüdiger Franke   |
+  | `Modelica.Thermal.FluidHeatFlow`      | Anton Haumer, Christian Kral                              |
+  | `Modelica.Thermal.HeatTransfer`       | Anton Haumer, Christian Kral                              |
+  | `Modelica.Math`                       | Martin Otter, Hans Olsson                                 |
+  | `Modelica.ComplexMath`                | Anton Haumer, Martin Otter, Christian Kral                |
+  | `Modelica.Utilities`                  | Martin Otter, Hans Olsson, Dag Brück                      |
+  | `Modelica.Constants`                  | Martin Otter, Hans Olsson                                 |
+  | `Modelica.Icons`                      | Christian Kral, Jakub Tobolar                             |
+  | `Modelica.SIunits`                    | Martin Otter, Christian Kral                              |
+  | `Modelica.Resources.C-Sources`        | Thomas Beutlich, Hans Olsson, Martin Sjölund              |
```

Comment:
This is simply wrong since there is no such library but the name refers to the C-sources

Reply:
Could be Modelica/Resources/C-Sources instead.

Reply:
but that would make the column unnecessarily wide again